src/main/scala/de/heikoseeberger/sbtfresh/Template.scala

```diff
@@ -313,6 +313,12 @@ private object Template {
        |}
        |""".stripMargin
 
+  def shellPromptWithGit: String =
+    """|import com.typesafe.sbt.SbtGit.GitCommand
```

Comment:
Hmm ... do you think we should get rid of `shell-prompt.sbt`? I got rid of all other files (like `project/Dependencies.scala` anyway. Then we could uns `enablePlugins(GitBranchPrompt)`.